packages/synapse-react-client/src/components/entity/page/title_bar/TitleBarProperties.integration.test.tsx

```diff
@@ -71,6 +71,25 @@ function useDoiAssociationOverride(doiAssociation: DoiAssociation | null) {
   )
 }
 
+function useGetFeatureFlagsOverride() {
+  server.use(
+    rest.get(
+      `${getEndpoint(
+        BackendDestinationEnum.PORTAL_ENDPOINT,
+      )}Portal/featureflags`,
+      (req, res, ctx) => {
+        return res(
+          ctx.status(200),
+          ctx.json({
+            'mock feature flag': true,
+          }),
+          ctx.set('Content-Type', 'application/json'), // Ensure header names are valid
+        )
+      },
+    ),
+  )
+}
```

Comment:
Can you put a reusable version of this in `mocks/msw/handlers/featureFlagHandlers.ts`, then use that in this file?